src/main/java/uk/gov/hmcts/reform/sscs/callback/handlers/DwpUploadResponseHandler.java

```diff
@@ -67,13 +68,13 @@ public void handle(CallbackType callbackType, Callback<SscsCaseData> callback) {
             log.info("updating to response received");
 
             String description = null;
-            if (!dwpFurtherInfo && !disputedDecision) {
+            if (!notDwpFurtherInfo && !notDisputedDecision) {
```

Comment:
not not and not not. It's better remove one negation and call the variable something your checking for